Javascript/index.js

```diff
@@ -47,4 +49,130 @@ const updateFormValues = () => {
+    description:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi Ut aliquip ex ea commodo consequat. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi.',
+    tech: ['html', 'Ruby on rail', 'css', 'Github'],
+  },
+  {
+    title: 'My Project',
+    image: './assets/images/Portfolio.svg',
+    description:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi Ut aliquip ex ea commodo consequat. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi.',
+    tech: ['html', 'Ruby on rail', 'css', 'Github'],
+  },
+  {
+    title: 'My Project',
+    image: './assets/images/Portfolio.svg',
+    description:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi Ut aliquip ex ea commodo consequat. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi.',
+    tech: ['html', 'Ruby on rail', 'css', 'Github'],
+  },
+  {
+    title: 'My Project',
+    image: './assets/images/Portfolio.svg',
+    description:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi Ut aliquip ex ea commodo consequat. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi.',
+    tech: ['html', 'Ruby on rail', 'css', 'Github'],
+  },
+];
+
+const btns = document.querySelectorAll('.buttons');
+const divProject = document.createElement('div');
+
+function addEventListenerList() {
+  for (let i = 0; i < btns.length; i += 1) {
+    btns[i].addEventListener('click', (event) => {
+      const projectId = event.target.id;
+      divProject.id = 'modal';
+      divProject.className = 'modal flex';
+      document.querySelector('.overal').style.display = 'flex';
+
+      let techHTML = '';
+      for (let l = 0; l < projects[i].tech.length; l += 1) {
+        techHTML += `<li class="modal-language">${projects[i].tech[l]}</li>`;
+      }
+
+      divProject.innerHTML = '<section class="modal-header flex">'
```

Comment:
- [ ] Your popup is not working as intended when the the `See Project` button is clicked. 
- There is no overlay displayed when the `See Project` button is triggered for the mobile version
- Your screen appears grayed out for desktop version and button click appears impossible
Kindly ensure to improve upon these issues mentioned

**Your current desktop  version:**
![image](https://user-images.githubusercontent.com/12745474/128611627-07049321-751b-4f62-89fb-25cb49d6ba84.png)